file.go

```diff
@@ -41,6 +42,16 @@ func RowLimit(n int) FileOption {
 	}
 }
 
+// ValueOnly treats all NULL values as meaningless and it will delete all NULL value cells,
+// before decode worksheet.xml. this option can save memory and time when parsing files
+// with a large number of NULL values. But it may also cause accidental injury,
+// because NULL maybe not really meaningless, it also could means empty or somethings else in the sheet.
```

Comment:
```suggestion
// because NULL may not really be meaningless.  Use with caution. 
```

Reply:
Thanks, I am also working hard to improve my English 😂. I have corrected it.

Reply:
Great, thanks.  I could have just corrected it myself, but it's always good to share.  As a native English speaker living in Germany, I'm very conscious that one can't improve in a second language if nobody ever corrects you.    I'll go ahead and merge this now.  Thanks again.